### PR TITLE
Add docs for the Localization module

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by sebastienros** [comment]
I am ok with adding doc comments to public APIs like the localization library, but in modules like this it doesn't make much sense. And it takes some time to review and merge. I would prefer we do it later.

What do others think?

File: src/OrchardCore.Modules/OrchardCore.Localization/GraphQL/Startup.cs [hunk 1]
```diff
@@ -4,9 +4,13 @@
 
 namespace OrchardCore.Localization.GraphQL
 {
+    /// <summary>
+    /// Represents an the localization module entry point for Graph QL.
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.Localization/GraphQL/Startup.cs]: an the localization => the localization

File: src/OrchardCore.Modules/OrchardCore.Localization/Permissions.cs [hunk 2]
```diff
@@ -5,15 +5,23 @@
 
 namespace OrchardCore.Localization
 {
+    /// <summary>
+    /// Represents a localization module permissions.
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.Localization/Permissions.cs]: a localization module permissions => the localization module permissions

**Comment by hishamco** [comment]
> but in modules like this it doesn't make much sense. 

Agree and disagree 😄 

Agree because  it takes some time to review and merge as you mentioned

Disagree because I saw many in the community trying to extend the current modules if they don't meet their needs, and there's no docs for such things

but I will leave it to the others ..

**Comment by agriffard** [comment]
Conflicts to solve.

**Pull request merged**